### PR TITLE
fix: deactivate landingpage for KR project

### PR DESCRIPTION
**Pull request by PierreDelpy**
In the KR project we don't have a logo yet, so we can't update the [landing rep](https://github.com/samply/bridgehead-landingpage) and thus have to deactivate the landing page

### CODE REVIEW

File: kr/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Krebsregister
+## Overview
+
+The **Krebsregister** project is designed to connect cancer registries across Germany by providing an overview of federated data. This allows for efficient querying and aggregation of data from multiple sources, empowering cancer research and healthcare analysis at a federal state level.
```
Comment (TKussel) [inline kr/README.md:4]:
```suggestion
The **Krebsregister** project is designed to pilot the connection between cancer registries across Germany by providing a federated overview of the available data. This allows for efficient querying and aggregation of data from multiple sources, empowering cancer research and healthcare analysis at a federal state level while ensuring privacy, data minimization, and data sovereignty.
```

File: kr/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+This project integrates several key modules to enable secure, scalable, and flexible data querying across federated registries. The core components used in this setup are:
+
+- Blaze FHIR Server: For managing FHIR-based medical data.
+- Focus: A tool for executing federated queries.
+- Beam Client: To connect the registry to the federated network.
```
Comment (TKussel) [inline kr/README.md:10]:
```suggestion
- Beam Proxy: To securely connect the registry to the federated network.
```

File: kr/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- oBDS2FHIR: For automated ETL (Extract, Transform, Load) of oBDS data into FHIR-compatible formats.
+
+## Modules in Use
+- Blaze FHIR Server
+- Focus
+- Beam Client
+- oBDS2FHIR-REST
```
Comment (TKussel) [inline kr/README.md:17]: A bit redundant with the previous paragraph. Maybe make it useful by providing the links to the repositories?

File: kr/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Installation
+
+To install the **Krebsregister** project, use the following command (in the directory ```/srv/docker/bridgehead```):
```
Comment (TKussel) [inline kr/README.md:21]:
```suggestion
To install the **Krebsregister** project bridgehead, use the following command (in the directory ```/srv/docker/bridgehead```):
```

File: kr/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+```bash
+
+./bridgehead install kr
+```
+This command will set up the required modules, including Blaze, Focus, Beam Client, and oBDS2FHIR, with all the necessary configurations in place.
```
Comment (TKussel) [inline kr/README.md:26]:
```suggestion
This command will interactively guide you through the installation process and set up the required modules, including Blaze, Focus, Beam Client, and oBDS2FHIR, with all the necessary configuration in place.
```

File: kr/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+Running the Project
```
Comment (TKussel) [inline kr/README.md:27]:
```suggestion
## Running the Project
```

File: kr/docker-compose.yml
Comment (TKussel) [inline kr/docker-compose.yml:1]: This is currently missing all the components, no? Or do you include them via modules by activating them in the config? I would prefer to see them here in the docker-compose.

**Review by TKussel — CHANGES_REQUESTED** [review]
Nice description of the process. In my opinion the required config repo could be added and the components are still missing from the docker-compose.yml

File: kr/README.md (hunk 3 above)
Comment (PierreDelpy) [reply kr/README.md:17]: Thx, I messed something up there. Commit is on the way

**Comment by TKussel** [comment]
> > Nice description of the process. In my opinion the required config repo could be added and the components are still missing from the docker-compose.yml
> 
> This should also be answered by this reply (or am I missing something?):
> 
> > the additional compose files are in the modules directory bridgehead/kr/modules/*.yml

I don't see this file in this PR, but in the module `lens-compose.yml` you are overriding the landing page again. I don't know how this interacts with this PR.

File: kr/README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+This command will interactively guide you through the installation process and set up the required modules, including Blaze, Focus, Beam Client, and oBDS2FHIR, with all the necessary configuration in place.
+## Running the Project
+
+Once installed, start the Krebsregister services with the following command:
+
+```bash
+
+systemctl start bridgehead@kr
+```
+This will activate the preconfigured modules and establish connections to the federated cancer registry network.
```
[the remark this answers is not in the document]
Comment (PierreDelpy) [reply kr/README.md:32]: fixed in https://github.com/samply/bridgehead/pull/234/commits/354733f7ea421d48bb4756fc3d6dfea696ff2bee

**Review by TKussel — APPROVED** [review]
The replicas thing is a great solution!

**Review by torbrenner — APPROVED** [review]

**Pull request merged**